examples/meta/src/regression/multiple_kernel_learning.sg

```diff
@@ -25,8 +25,8 @@ combined_kernel.init(features_train, features_train)
 #![create_combined_train]
 
 #![train_mkl]
-SVM binary_svm_solver = as_svm(machine("SVRLight"))
-Machine mkl = machine("MKLRegression", svm=binary_svm_solver, kernel=combined_kernel, labels=labels_train)
+SVM binary_svm_solver = machine("SVRLight")
```

Comment:
for some reason if `as_svm` is being used here the ref counting is broken :S

Reply:
broken in which way? too high or too low?